spec/features/event_spec.rb

```diff
@@ -135,12 +135,12 @@
       end
     end
 
-    it "should not allow dates in the past" do
+    it "should allow dates in the past" do
```

Comment:
Wow, didn't think of that. I like the idea, would changing the test action to editing an existing event and saving it without errors be what you suggested?